presto-main/src/main/java/io/prestosql/SystemSessionProperties.java

```diff
@@ -125,6 +125,8 @@
     public static final String DYNAMIC_FILTERING_MAX_PER_DRIVER_ROW_COUNT = "dynamic_filtering_max_per_driver_row_count";
     public static final String DYNAMIC_FILTERING_MAX_PER_DRIVER_SIZE = "dynamic_filtering_max_per_driver_size";
     public static final String IGNORE_DOWNSTREAM_PREFERENCES = "ignore_downstream_preferences";
+    public static final String REQUIRED_WORKERS = "required_workers";
+    public static final String REQUIRED_WORKERS_MAX_WAIT = "required_workers_max_wait";
```

Comment:
I don't mind changing it, however this would make the session property names slightly inconsistent with the existing config property names `query-manager.required-workers`, `query-manager.required-workers-max-wait`. Would that be ok ?

Reply:
That's a good point, though I was thinking it's better to have them consistent with other session properties like `hash_partition_count` and `query_max_execution_time`, hence the `_count` and `_time` suffixes. Session properties are more "user facing" than config properties, so consistency is good.

Reply:
Please go with the names Dain posted.

Reply:
Done, changed to `required_workers_count` and `required_workers_max_wait_time`